### DISCUSSION TITLE
Documentation Updates 🎉

### DISCUSSION THREAD

**Discussion by 2percentsilk** [community/community#3953] [Codespaces]
Exciting news to share - we have made lots of changes to our documentation.
Here's a quick summary of our changes so that you can check them out and share any feedback you have with us. 📄 
[Landing page](https://docs.github.com/en/codespaces) ✈️ : The landing page aims at providing an overview on Codespaces with a quick 2 min video and a place to find links to various documentation topics that you might be looking for as you explore Codespaces! 
[Quickstart](https://docs.github.com/en/codespaces/quickstart) ⏲️:  To explore some of the essential features available to you within a Codespace, you can follow  the quickstart doc to fire up a codespace with a sample repository in 5 mins or less!
[Getting started guides](https://docs.github.com/en/codespaces/getting-started-with-codespaces) 💡: Once you have a basic understanding on what Codespaces are and how they work, you can proceed to the guides that help you set up and configure specific repositories such as Node.js, C#, Java, Python for Codespaces access. Let us know which other languages would you like to see added to these guides.
[Managing Codespaces for your organization](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization) 👥 : We have created a separate section to help answer any of your questions as you set up and manage Codespaces for your organization. This section includes more information on managing secrets, access & security, user permissions and audit logs for your org.
Lastly, you can dive deeper into the [Setting up your Codespace](https://docs.github.com/en/codespaces/setting-up-your-codespace) and [Developing in a Codespace](https://docs.github.com/en/codespaces/developing-in-codespaces) sections for guidance on how to configure and use Codespaces for your org repositories.

**Comment by tabaco26** [comment]
API

**Comment by Br0w07** [comment]
Update for new version system
